package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@condenast/cn-react-slick",
-  "version": "0.14.5",
+  "version": "0.14.5-cn.1",
```

Comment:
this is to keep the version in line with the canonical, essentially?

Later:
can we open some PRs against the source to try and (eventually) not need a fork?